devtools/cli/src/main/java/io/quarkus/cli/common/BuildOptions.java

```diff
@@ -13,10 +13,17 @@
     @CommandLine.Option(order = 5, names = { "--offline" }, description = "Work offline.", defaultValue = "false")
     public boolean offline = false;
 
-    @CommandLine.Option(order = 6, names = { "--tests" }, description = "Run tests.", negatable = true)
-    public boolean runTests = true;
+    @CommandLine.Option(order = 6, names = {
+            "--no-tests" }, description = "Run tests.", negatable = true, defaultValue = "false")
+    public boolean skipTests = false;
```

Comment:
i thought you didn't like double negations :) 

had exepcted to still be called runTests.

Reply:
the way it is actually set, it ends up matching the positive skipTests in usage.